scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -455,3 +455,25 @@ def test_from_euler_extrinsic_rotation_313():
         [0.91855865, -0.30618622, -0.250000],
         [0.35355339, 0.35355339, 0.8660254]
     ]))
+
+
+def test_as_euler_extrinsic_rotation():
+    np.random.seed(0)
+    angles = np.random.uniform(low=-np.pi, high=np.pi, size=(6, 3))
+    rot_obj = Rotation.from_euler('zxy', angles)
+
+    estimates = rot_obj.as_euler('zxy')
+    est_obj = Rotation.from_euler('zxy', estimates)
```

Comment:
OK, it makes sense to me, but isn't it better to compare angles directly? Preferably we want to see, that the ranges our algorithm selects are easy to understand and intuitive. I.e if used `roll=10, pitch=20, yaw=150` we want to see the same angles back, not some bizarre transformation of them. You see my idea.

Reply:
You're right. I will include some extra tests for that as well.
Since I was initializing random angles here, the returned angles may not be the same due to adjustments. Hence the dcm comparisons.